config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -56,7 +56,7 @@
 Rails.application.config.action_mailer.smtp_timeout = 5
 
 # Automatically infer `inverse_of` for associations with a scope.
-# Rails.application.config.active_record.automatic_scope_inversing = true
+Rails.application.config.active_record.automatic_scope_inversing = true
```

Comment:
This reduces calls to DB. A potential performance boost but we can remove it if it causes any problem.